bin/plugin/commands/build-plugins.js

```diff
@@ -0,0 +1,296 @@
+/**
+ * External dependencies
+ */
+const fs = require( 'fs-extra' );
+const path = require( 'path' );
+const glob = require( 'fast-glob' );
+
+/**
+ * Internal dependencies
+ */
+const { log, formats } = require( '../lib/logger' );
+
+exports.options = [];
+
+/**
+ * Command for copying the files required for the standalone plugin.
+ */
+exports.handler = async () => {
+	const pluginsFile = path.join( '.', 'plugins.json' );
+	fs.readFile( pluginsFile, 'utf8', ( err, jsonString ) => {
+		if ( err ) {
+			log(
+				formats.error(
+					`\nError reading file from disk: "${ err }"\n\n`
+				)
+			);
+		}
+
+		try {
+			const modules = JSON.parse( jsonString );
+			for ( const key in modules ) {
+				const pluginKey = key;
```

Comment:
Both `key` and `pluginKey` are not really describing what this value is, as it is the module directory. Also we don't need two variables for this.
```suggestion
			for ( const moduleDir in modules ) {
```